modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/rest/RestExecutor.java

```diff
@@ -47,6 +48,9 @@
     /** */
     private static final IgniteLogger log = new Slf4jLogger(LoggerFactory.getLogger(RestExecutor.class));
 
+    /** */
+    private static final long REQUEST_TIMEOUT = MINUTES.toMillis(3L);
```

Comment:
May be this can be a system property with fallback to default value?
This can be useful for tests.

Reply:
Done